### PR TITLE
[FEATURE] Disable api in production mode

### PR DESCRIPTION
**Pull request by stevenzeiler**
Add basic api index as /
Removes html home page in favor of json only api

### CODE REVIEW

**Comment by stevenzeiler** [comment]
Setting `NODE_ENV=production` will disable all the api routes for the time being, so we can test our deployment infrastructure without exposing unfinished features.

File: api/controllers/IndexController.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+/**
+ * IndexController
+ *
+ * @description :: Server-side logic for managing Indexs
```
Comment (wilsonianb) [inline api/controllers/IndexController.js]: `Indexs` -> `Indexes`/`Indices`?

**Comment by stevenzeiler** [comment]
Later we can use the same "policies" mechanism to restrict POST access to authorized servers using access tokens.

File: api/controllers/IndexController.js (hunk 1 above)
Comment (stevenzeiler) [inline api/controllers/IndexController.js]: Oh yep, a search-and-replace mishap.

**Comment by stevenzeiler** [comment]
+r

File: api/policies/disableProduction.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+  // User is allowed, proceed to the next policy, 
+  // or if this is the last policy, the controller
+  if (process.env.NODE_ENV === 'production') {
+    return res.forbidden('You are not permitted to perform this action.');
+  } else {
+    return next()
+  }
+  // User is not allowed
+  // (default res.forbidden() behavior can be overridden in `config/403.js`)
```
Comment (wilsonianb) [inline api/policies/disableProduction.js]: The comments in this function seem misplaced.

**Comment by wilsonianb** [comment]
Did you mean to include `config/.routes.js.swp`?

File: config/policies.js [hunk 3]
```diff
@@ -26,7 +26,7 @@ module.exports.policies = {
   *                                                                          *
   ***************************************************************************/
 
-  // '*': true,
+  '*': 'disableProduction',
```
Comment (wilsonianb) [inline config/policies.js]: Remove comma

**Comment by stevenzeiler** [comment]
No I thought I had deleted the .swp file

**Comment by wilsonianb** [comment]
You're right. I must be colorblind.

**Comment by wilsonianb** [comment]
It works. I get the Forbidden page with `NODE_ENV=production`
LGTM :+1:

**Pull request merged**